cmake/portaudio-2.0.pc.in

```diff
@@ -0,0 +1,13 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: PortAudio
+Description: Portable audio I/O
+Requires.private: @PKGCONFIG_REQUIRES@
+Version: @CMAKE_PROJECT_VERSION_MAJOR@
+
+Libs: -L${libdir} -lportaudio
+Libs.private: @PKGCONFIG_LDFLAGS@
```

Comment:
Note that the use of the `Requires.private` and `Libs.private` fields is different from the pkg-config file generated by CMake in master as well as the pkg-config file generated by autotools. This is [recommended by the pkg-config maintainer](https://people.freedesktop.org/~dbn/pkg-config-guide.html#faq):

> My library z installs header files which include libx headers. What do I put in my z.pc file?
If the x library has pkg-config support, add it to the Requires.private field. If it does not, augment the Cflags field with the necessary compiler flags for using the libx headers. In either case, pkg-config will output the compiler flags when --static is used or not.

> My library z uses libx internally, but does not expose libx data types in its public API. What do I put in my z.pc file?
Again, add the module to Requires.private if it supports pkg-config. In this case, the compiler flags will be emitted unnecessarily, but it ensures that the linker flags will be present when linking statically. If libx does not support pkg-config, add the necessary linker flags to Libs.private.